guide/matrix-vs-discord.md

```diff
@@ -112,10 +121,28 @@ Note that Matrix does not (and cannot, due to its decentralized nature) paywall
 
 [^2]: For Discord "servers" and for Matrix rooms.
 
-[^3]: Note that Matrix has no concept of "friends" or "contacts," although the DM list can serve the same purpose.
+[^3]: Note that Matrix has no concept of "friends" or "contacts" per se, although the DM list can serve the same purpose. However, user-based contact ignoring exists on Matrix, using two different methods. The former method prevents any and all messages from reaching the recipient, and the latter method using policy rooms hides them client-side by interpreting an ignore list. The latter method is currently only available in Element as a labs feature.
```

Comment:
> The former method prevents any and all messages from reaching the recipient,

Only encrypted ones (key withheld). I can still see unencrypted messages of people who've blocked me.

Reply:
Ignores are receiver-initiated.